src/main/java/domain/LottoNumbers.java

```diff
@@ -1,17 +1,46 @@
 package domain;
 
-import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 
 public class LottoNumbers {
 
-    private List<Integer> lottoNumbers = new ArrayList<>();
+    private static final int MIN_LOTTO_NUM = 1;
+    private static final int MAX_LOTTO_NUM = 45;
+
+    private static final int LOTTO_NUM_COUNT = 6;
+
+
+    private List<Integer> lottoNumbers;
 
     public LottoNumbers(PickNumberStrategy pickNumberStrategy) {
         lottoNumbers = pickNumberStrategy.makeLottoNumber();
+        validate();
+    }
+
+    private void validate() {
+        if (!lottoNumbers.stream().allMatch(number -> (number <= MAX_LOTTO_NUM && number >= MIN_LOTTO_NUM))) {
+            throw new IllegalArgumentException("1과 45사이의 값만 들어올 수 있습니다.");
+        }
+        if (lottoNumbers.size() != LOTTO_NUM_COUNT) {
+            throw new IllegalArgumentException("로또의 사이즈는 6개만 가능합니다.");
```

Comment:
```suggestion
            throw new IllegalArgumentException("로또의 사이즈는 " + LOTTO_NUM_COUNT  + "개만 가능합니다.");
```

상수를 활용하면 조건이 변경 됐을 때 메시지를 신경쓰지 않을 수 있겠네요!

Reply:
네네 해당 부분도 수정하였습니다 👍